File-Formats/XlsIO/Worksheet-Cells-Manipulation.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Worksheet Cells Manipulation | Syncfusion
-description: This section illustrates about cells manipulation in Excel worksheet using Syncfusion XlsIO (a .NET Excel library).
+title: Worksheet Cells Manipulation in Fileformat PDF | Syncfusion
```

Comment:
Change like this - Worksheet Cells Manipulation in File Formats XlsIO Control | Syncfusion

Reply:
Resolved